.github/workflows/submission.yml

```diff
@@ -39,7 +39,7 @@ jobs:
     - name: Update reactions badge
       run: |
         cd "${GITHUB_WORKSPACE}"
-        python badges/reactions.py --root=data --output=badges/reactions.svg
+        python ord-schema/badges/reactions.py --root=data --output=badges/reactions.svg
```

Comment:
output needs to be `ord-schema/badges/reactions.svg`?

Reply:
No, I'm keeping the output SVG in ord-data; just the script to generate it lives in ord-schema. Does that make sense?

Reply:
Yes, that seems reasonable